src/software/ai/hl/stp/tactic/attacker/attacker_fsm.cpp

```diff
@@ -103,3 +103,69 @@ bool AttackerFSM::shouldKick(const Update& event)
     // otherwise check for shot or pass committed
     return event.control_params.pass_committed || event.control_params.shot;
 }
+
+void AttackerFSM::alignToBall(const Update& event,
+                              boost::sml::back::process<MoveFSM::Update> processEvent)
+{
+    Vector ball_to_center_vec =
+        Vector(0, 0) - event.common.world.ball().position().toVector();
+    // We want the kicker to get into position behind the ball facing the center
+    // of the field
+    MoveFSM::ControlParams control_params{
+        .destination = event.common.world.ball().position() -
+                       ball_to_center_vec.normalize(ROBOT_MAX_RADIUS_METERS * 2),
+        .final_orientation      = ball_to_center_vec.orientation(),
+        .final_speed            = 0.0,
+        .dribbler_mode          = TbotsProto::DribblerMode::OFF,
+        .ball_collision_type    = TbotsProto::BallCollisionType::AVOID,
+        .auto_chip_or_kick      = AutoChipOrKick{AutoChipOrKickMode::OFF, 0},
+        .max_allowed_speed_mode = TbotsProto::MaxAllowedSpeedMode::PHYSICAL_LIMIT,
+        .target_spin_rev_per_s  = 0.0};
+
+    processEvent(MoveFSM::Update(control_params, event.common));
+}
```

Comment:
is there a reason why we're not using GetBehindBall FSM

Reply:
get behind ball puts the robot in a cone, whereas here we're putting it on a point. I don't think cone is what we want for this quick alignment tactic